### PR TITLE
Fix issues with config and launch

### PR DESCRIPTION
**Pull request by tabiosg**
#8 was buggy so this is made. Made by @sashreek1.

### CODE REVIEW

**Comment by tabiosg** [comment]
Lesson learned, #8  was only "buggy" because science_ros was not merged with config_files.

**Comment by tabiosg** [comment]
Once again, I'll review it tomorrow after I watch my daily dose of anime watching

File: config/science_board_config.yaml [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-  white_led: 5
-serial:
-  baudrate: 38400
-  timeout: 0
-info:
-  max_error_count: 20
-  sleep: 0.01
-  uart_transmit_msg_len: 30
+led_map:
+  "red": 0
```
Comment (tabiosg) [inline config/science_board_config.yaml:2]: i seriously don't understand why this caused issues smmhhh

File: src/esw/science_board/science_board.py [hunk 2]
```diff
@@ -20,14 +22,19 @@
 class ScienceBridge():
     """ScienceBridge class"""
     def __init__(self):
-        with open('config.yml', 'r') as file:
-            config = yaml.safe_load(file)
-        self.baudrate = config['serial']['baudrate']
+
+        """with open(rospy.get_param("/science_config_file"), 'r') as file:
```
Comment (tabiosg) [inline src/esw/science_board/science_board.py]: just delete comment

File: config/science_board_config.yaml [hunk 3]
```diff
@@ -1,21 +1,26 @@
-led_map:
```
Comment (tabiosg) [inline config/science_board_config.yaml:1]: ok so i guess now we just load in all this info into ros param server. what happens if two files use the same name? should we make the names more descriptive (e.g. info)

File: srv/ChangeAutonLEDState.srv [hunk 4]
```diff
@@ -1,3 +1,3 @@
-string color
```
Comment (tabiosg) [inline srv/ChangeAutonLEDState.srv:1]: i literally have no idea what differences were made between these files

**Review by tabiosg — COMMENTED** [review]
Ok I got bored so i reviewed anyway, plz respond to the all conversations before i merge @sashreek1

File: config/science_board_config.yaml (hunk 3 above)
Comment (sashreek1) [reply config/science_board_config.yaml:1]: I don't understand what you're asking here. You want to know what happens if two files use "led_map" ? Which name do you want to change to "info"?

File: config/science_board_config.yaml (hunk 1 above)
Comment (sashreek1) [reply config/science_board_config.yaml:2]: Cause the keyword had to be a string lol

File: config/science_board_config.yaml (hunk 3 above)
Comment (tabiosg) [reply config/science_board_config.yaml:1]: So like two different yaml files might interfere if they both use something as common as /info/

Comment (tabiosg) [reply config/science_board_config.yaml:1]: So like info/sleep can easily be declared by a different configuration file

Comment (sashreek1) [reply config/science_board_config.yaml:1]: RIght, yeah we could name it something more descriptive

File: config/science_board_config.yaml (hunk 1 above)
Comment (tabiosg) [reply config/science_board_config.yaml:2]: That's what we call CRINGE in the industry

File: src/esw/science_board/science_board.py (hunk 2 above)
Comment (tabiosg) [reply src/esw/science_board/science_board.py]: yea general note is not leave ghost comments, just delete them

**Comment by tabiosg** [comment]
Looks good, just delete ghost comments next time

**Pull request merged**